fuzz/fuzz.go

```diff
@@ -0,0 +1,51 @@
+package fuzz
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+// Test runs fuzz once for each input in corpus.
+func Test(t *testing.T, fuzz func([]byte) int) {
+	t.Helper()
+
+	err := filepath.Walk("testdata", func(path string, info os.FileInfo, err error) error {
+		t.Helper()
+
+		if err != nil {
+			return err
+		}
+
+		if info.IsDir() && path == "suppressions" {
```

Comment:
Could you add a comment here to explain why we are skipping this directory?

Reply:
I’ve updated walkFn so that it only visits regular files in corpus and crashers directories.